docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-jira-service-management.md

```diff
@@ -0,0 +1,66 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-jira-service-management/
+description: Configure the Jira Service Management integration to receive notifications when your alerts are firing
+keywords:
+  - grafana
+  - alerting
+  - Jira
+  - Jira Service Management
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Jira Service Management
+title: Configure Jira Service Management for Alerting
+weight: 121
+---
+
+# Configure Jira Service Management for Alerting
+
+Use the Jira integration in a contact point to create issues in your Jira instance when alerts fire. The integration supports both Jira Cloud and Jira Server/Data Center installations.
+
+## Before you begin
+
+Before you begin, ensure you have the following:
+
+- A Jira instance (Cloud, Server, or Data Center)
+- API access credentials for Jira
+- Appropriate permissions to create issues in your target Jira project
+
+## Configure Jira for a contact point
+
+To create a contact point with a Jira integration, complete the following steps:
+
+1. Navigate to **Alerts & IRM** -> **Alerting** -> **Contact points**.
+2. Click **+ Add contact point**.
+3. Enter a name for the contact point.
+4. From the **Integration** list, select **Jira**.
+5. Set up the required [settings](#jira-settings) for your Jira configuration.
+6. Click **Save contact point**.
+
```

Comment:
For [webhook](https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier/) and [SNS](https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-amazon-sns/) , we have recently included the following call-to-action after saving the contact point.

> For more details on contact points, including how to test them and enable notifications, refer to [Configure contact points](https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/).

You might find a better way to point this out, I believe referencing these instructions in these docs are helpful to indicate how to test the contact point.